packages/chakra-ui-docs/utils/index.js

```diff
@@ -17,3 +17,12 @@ export const removeHyphenFromString = (hyphenatedString) => {
   const str = hyphenatedString.split('-')
   return str.join(' ')
 }
+
+export const findNextAndPrevRoute = (path, routes) => {
+  const currentRouteIndex = routes.map(route => route.path).indexOf(path)
+
+  const nextPageLink = routes[currentRouteIndex + 1]
+  const prevPageLink = routes[currentRouteIndex - 1]
+
+  return { prev: prevPageLink ? prevPageLink.path : '', next: nextPageLink ? nextPageLink.path : '' }
```

Comment:
Nice work here. We should also capitalize the path name. For example, `accordion` -> `Accordion`

Reply:
I'm only using the returned path to navigate to the next route. If I were to change it, that would mean we would have a route like `/Constraint-based-design`.

Just confirming if this is what we want? @codebender828

Reply:
Oh I think I might have been unclear. I meant for the text that's rendered in the link.

Reply:
See this image for reference:

![Chakra UI Vue Docs links](https://user-images.githubusercontent.com/21237954/83329303-421cf700-a2bb-11ea-9be5-a7e9fd5ff1f2.png)

In order to get actual readable text back, we might need to change the values from kebab-case to sentence case before capitalizing it.

Reply:
Oh, that makes sense, I just had `Prev` and `Next` in there before, thanks for clearing that up 👍🏽

Reply:
In your current commit, the text is too close to the bottom of the page. Also, a border around the links might make it stand out more

Reply:
In your current commit, the text is too close to the bottom of the page. Also, a border around the links might make it stand out more